plugin-server/src/cdp/templates/_destinations/linear/linear.template.ts

```diff
@@ -0,0 +1,80 @@
+import { HogFunctionTemplate, SUB_TEMPLATE_COMMON } from '../../types'
+
+export const template: HogFunctionTemplate = {
+    status: 'alpha',
```

Comment:
Experimental destinations are hidden by default and can only be installed with the direct link. This should probably be `beta` instead

Reply:
I actually made it `alpha` because I wanted to test the bug I mentioned above about the Hog templating. I'm also working on a GitHub integration and some more native integrations with issues so will release them all together